Configuration/python/Color.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
There are more instances of these that could be removed now with this module:
```
Configuration/python/ProgressIndicator.py
AnaTools/interface/DataFormat.h
AnaTools/scripts/setupFramework.py
```
Good idea, please clean those 3 up too.

Reply:
I imported `Color.py` in `ProgressIndicator.py`.

But `DataFormat.h` is C++, not Python, so I can't import there. And `setupFramework.py` is typically run before doing `scram b`, which sets up the links in `$CMSSW_BASE/python/`, so it cannot import things in the framework, unless we want the user to run `scram b` first, then `setupFramework.py`, then run `scram b` again. Which I don't think we do.

Reply:
Whoops, fair points.